### PR TITLE
Bump mod versions and fix standalone server

### PR DESCRIPTION
**Pull request by nicholasgrose**
_This is a copy of #291 except that I am splitting it off from my main dev branch to support continued experimentation. I'll simply cherry pick fixes into it._

This contains several notable changes:
1. Added .idea to ignore files.
2. Updates packwiz and adds a linux executable for it
3. Updates many mods (via packwiz update command), including Fabric

Notable exceptions to the updates:
1. Iris was updated from 1.6.6 to 1.6.10, rather than 1.6.11. 1.6.11 is unable to initialize, and my investigation has been unable to turn up why this is. It appears to be something to do with mixins, but more research is needed.
2. Left REI at the current version because newer versions experience a crash when viewing some recipes. (Reproduced by pressing "U" on normal crafting tables. The crash might have been related to YTTR?)
3. Updated to a nightly build of CreateBigCannons due to [this bug](https://github.com/Cannoneers-of-Create/CreateBigCannons/issues/521)
4. Updated to a self-compiled nightly build of iris-flywheel-compat due to [this commit](https://github.com/leon-o/iris-flw-compat/commit/e19c58eb0ac27d2b41c66427f56956a876a8b298) that fixes it for Create 0.5.1c+.
5. Updated to a self-compiled, self-modified build of Automobility to address [this bug](https://github.com/FoundationGames/Automobility/issues/85). This bug prevents dedicated servers from running, and it was an absolute pain to fix. The steps taken to resolve it involved recompiling an older version of [ARRP (MPL-2.0 License)](https://github.com/Devan-Kerman/ARRP) that supports 1.18.2 (0.5.5 no longer exists on Maven) and making the following change to the source code of Automobility (MIT License):
image

### CODE REVIEW

File: mods/iris-flywheel-compat-1.18.2-0.2.0.jar
Comment (Erdragh) [inline mods/iris-flywheel-compat-1.18.2-0.2.0.jar:1]: I already added this in my ponder PR (#278). But I'll just merge your changes into my branch once it's merged.

File: mods/modernfix.pw.toml
Comment (Erdragh) [inline mods/modernfix.pw.toml:1]: I'm not sure whether there was a specific reason for being on a certain modernfix version. If I recall correctly, there were some issues with CC: Tweaked monitor rendering. I'd need a confirmation from @Laskyyy tho.

**Review by Erdragh — COMMENTED** [review]
Can you justify adding the .idea contents? Especially for specific plugins and deployment?

File: mods/modernfix.pw.toml
Comment (Erdragh) [reply mods/modernfix.pw.toml:1]: I just tested your branch, the only visual bug with monitors seems to be the one caused by iris, which we have to live with...

**Comment by nicholasgrose** [comment]
> Can you justify adding the .idea contents? Especially for specific plugins and deployment?

Not particularly. that was just a lack of attention. That's now been reverted and it is just gitignored.

**Review by Erdragh — APPROVED** [review]

**Comment by Erdragh** [comment]
Hey @nicholasgrose in case you're still around, we're having another server side issue with Automobility, namely that it crashes whenever you place anything from the mod... Are your changes to Automobility pushed somewhere?

**Comment by nicholasgrose** [comment]
Unfortunately, they are not currently pushed, but I can do so next week, when I'm back from travelling.

> Hey @nicholasgrose in case you're still around, we're having another server side issue with Automobility, namely that it crashes whenever you place anything from the mod... Are your changes to Automobility pushed somewhere?

**Comment by Erdragh** [comment]
Actually, don't worry about it, I fixed it already. Had to remove some more `@Environment` annotations. Thanks for thoroughly documenting what you did!

**Comment by nicholasgrose** [comment]
> Actually, don't worry about it, I fixed it already. Had to remove some more `@Environment` annotations. Thanks for thoroughly documenting what you did!

I'm glad to hear it!

**Comment by Erdragh** [comment]
Another thing, the Issue link for Big Cannons leads to a 404, can you recall what that was about? @nicholasgrose

**Comment by nicholasgrose** [comment]
> Another thing, the Issue link for Big Cannons leads to a 404, can you recall what that was about? @nicholasgrose 

I don't. However, I do know that the most recent patch version came out not long after I picked up the nightly. It would have whatever the fix was included.

**Comment by Erdragh** [comment]
Ah, great. Also, archive.org seems to actually have had a capture of that issue, once: https://web.archive.org/web/20240421151437/https://github.com/Cannoneers-of-Create/CreateBigCannons/issues/521

**Pull request merged**